### PR TITLE
chg: Initial commit to rename bro -> zeek

### PR DESCRIPTION
**Pull request by Rafiot**
It isn't mergeable as-is because it breaks current URLs using bro. 

I think the proper way to support that is to keep the bro ones as alias of zeek.

### CODE REVIEW

File: app/Console/Command/EventShell.php [hunk 1]
```diff
@@ -376,29 +376,29 @@ public function cachenids() {
...
 		$this->Job->id = $id;
 		$this->Job->saveField('progress', 1);
-		App::uses('BroExport', 'Export');
-		$export = new BroExport();
+		App::uses('ZeekExport', 'Export');
+		$export = new ZeekExport();
 		$types = array_keys($export->mispTypes);
 		$typeCount = count($types);
-		$dir = new Folder(APP . DS . '/tmp/cached_exports/bro', true, 0750);
+		$dir = new Folder(APP . DS . '/tmp/cached_exports/zeek', true, 0750);
```
Comment (Rafiot) [inline app/Console/Command/EventShell.php:392]: Does it break things?

File: app/Controller/EventsController.php [hunk 2]
```diff
@@ -4285,9 +4285,9 @@ public function exportChoice($id)
                     'requiresPublished' => true,
                     'checkbox' => false,
             ),
-            'bro' => array(
-                    'url' => '/attributes/bro/download/all/false/' . $id,
-                    'text' => 'Download Bro rules',
+            'zeek' => array(
+                    'url' => '/attributes/zeek/download/all/false/' . $id,
```
Comment (Rafiot) [inline app/Controller/EventsController.php:4289]: Alias needed.

File: app/View/Events/automation.ctp [hunk 3]
```diff
@@ -105,19 +105,19 @@
...
-    <p><?php echo __('An export of all attributes of a specific bro type to a formatted plain text file. By default only published and IDS flagged attributes are exported.');?></p>
-    <p><?php echo __('You can configure your tools to automatically download a file one of the Bro types.');?></p>
+    <h3><?php echo __('Zeek (Bro) IDS export');?></h3>
+    <p><?php echo __('An export of all attributes of a specific zeek type to a formatted plain text file. By default only published and IDS flagged attributes are exported.');?></p>
+    <p><?php echo __('You can configure your tools to automatically download a file one of the Zeek (Bro) types.');?></p>
     <pre><?php
-        foreach (array_keys($broTypes) as $broType) {
-            echo $baseurl.'/attributes/bro/download/'.$broType . "\n";
+        foreach (array_keys($zeekTypes) as $zeekType) {
+            echo $baseurl.'/attributes/zeek/download/'.$zeekType . "\n";
```
Comment (Rafiot) [inline app/View/Events/automation.ctp:113]: Alias needed.

File: app/View/Events/automation.ctp [hunk 4]
```diff
@@ -105,19 +105,19 @@
...
         }
     ?></pre>
     <p><?php echo __('To restrict the results by tags, use the usual syntax. Please be aware the colons (:) cannot be used in the tag search. Use semicolons instead (the search will automatically search for colons instead). To get ip values from events tagged tag1 but not tag2 use');?>:</p>
-    <pre><?php echo $baseurl.'/attributes/bro/download/ip/tag1&&!tag2'; ?></pre>
+    <pre><?php echo $baseurl.'/attributes/zeek/download/ip/tag1&&!tag2'; ?></pre>
```
Comment (Rafiot) [inline app/View/Events/automation.ctp:117]: Alias needed.

File: app/View/Events/automation.ctp [hunk 5]
```diff
@@ -105,19 +105,19 @@
...
 
-    <p><?php echo __('It is possible to restrict the bro exports on based on a set of filters. POST a JSON object or an XML at the Bro API to filter the results.');?></p>
-    <pre><?php echo $baseurl.'/attributes/bro/download'; ?></pre>
+    <p><?php echo __('It is possible to restrict the Zeek (Bro) exports on based on a set of filters. POST a JSON object or an XML at the Zeek API to filter the results.');?></p>
+    <pre><?php echo $baseurl.'/attributes/zeek/download'; ?></pre>
```
Comment (Rafiot) [inline app/View/Events/automation.ctp:120]: Alias needed.

File: app/View/Events/automation.ctp [hunk 6]
```diff
@@ -135,10 +135,10 @@
     ?></pre>
     <code>&lt;request&gt;&lt;type&gt;ip&lt;/type&gt;&lt;eventid&gt;!51&lt;/eventid&gt;&lt;eventid&gt;!62&lt;/eventid&gt;&lt;withAttachment&gt;false&lt;/withAttachment&gt;&lt;tags&gt;APT1&lt;/tags&gt;&lt;tags&gt;!OSINT&lt;/tags&gt;&lt;from&gt;false&lt;/from&gt;&lt;to&gt;2015-02-15&lt;/to&gt;&lt;/request&gt;</code><br /><br />
     <p><?php echo __('Alternatively, it is also possible to pass the filters via the parameters in the URL, though it is highly advised to use POST requests with JSON objects instead. The format is as described below');?>:</p>
-    <pre><?php echo $baseurl.'/attributes/bro/download/[type]/[tags]/[event_id]/[allowNonIDS]/[from]/[to]/[last]'; ?></pre>
-    <b>type</b>: <?php echo __('The Bro type, any valid Bro type is accepted. The mapping between Bro and MISP types is as follows');?>:<br />
+    <pre><?php echo $baseurl.'/attributes/zeek/download/[type]/[tags]/[event_id]/[allowNonIDS]/[from]/[to]/[last]'; ?></pre>
```
Comment (Rafiot) [inline app/View/Events/automation.ctp:138]: Alias needed.

File: app/View/Events/legacy_automation.ctp [hunk 7]
```diff
@@ -208,19 +208,19 @@
...
-    <p><?php echo __('An export of all attributes of a specific bro type to a formatted plain text file. By default only published and IDS flagged attributes are exported.');?></p>
-    <p><?php echo __('You can configure your tools to automatically download a file one of the Bro types.');?></p>
+    <h3><?php echo __('Zeek (Bro) IDS export');?></h3>
+    <p><?php echo __('An export of all attributes of a specific zeek type to a formatted plain text file. By default only published and IDS flagged attributes are exported.');?></p>
+    <p><?php echo __('You can configure your tools to automatically download a file one of the Zeek types.');?></p>
     <pre><?php
-        foreach (array_keys($broTypes) as $broType) {
-            echo $baseurl.'/attributes/bro/download/'.$broType . "\n";
+        foreach (array_keys($zeekTypes) as $zeekType) {
+            echo $baseurl.'/attributes/zeek/download/'.$zeekType . "\n";
```
Comment (Rafiot) [inline app/View/Events/legacy_automation.ctp:216]: Alias needed.

File: app/View/Events/legacy_automation.ctp [hunk 8]
```diff
@@ -208,19 +208,19 @@
...
         }
     ?></pre>
     <p><?php echo __('To restrict the results by tags, use the usual syntax. Please be aware the colons (:) cannot be used in the tag search. Use semicolons instead (the search will automatically search for colons instead). To get ip values from events tagged tag1 but not tag2 use');?>:</p>
-    <pre><?php echo $baseurl.'/attributes/bro/download/ip/tag1&&!tag2'; ?></pre>
+    <pre><?php echo $baseurl.'/attributes/zeek/download/ip/tag1&&!tag2'; ?></pre>
 
-    <p><?php echo __('It is possible to restrict the bro exports on based on a set of filters. POST a JSON object or an XML at the Bro API to filter the results.');?></p>
-    <pre><?php echo $baseurl.'/attributes/bro/download'; ?></pre>
+    <p><?php echo __('It is possible to restrict the zeek exports on based on a set of filters. POST a JSON object or an XML at the Zeek API to filter the results.');?></p>
+    <pre><?php echo $baseurl.'/attributes/zeek/download'; ?></pre>
```
Comment (Rafiot) [inline app/View/Events/legacy_automation.ctp:223]: Alias needed.

File: app/View/Events/legacy_automation.ctp [hunk 9]
```diff
@@ -208,19 +208,19 @@
...
     <pre><?php
-        foreach (array_keys($broTypes) as $broType) {
-            echo $baseurl.'/attributes/bro/download/'.$broType . "\n";
+        foreach (array_keys($zeekTypes) as $zeekType) {
+            echo $baseurl.'/attributes/zeek/download/'.$zeekType . "\n";
         }
     ?></pre>
     <p><?php echo __('To restrict the results by tags, use the usual syntax. Please be aware the colons (:) cannot be used in the tag search. Use semicolons instead (the search will automatically search for colons instead). To get ip values from events tagged tag1 but not tag2 use');?>:</p>
-    <pre><?php echo $baseurl.'/attributes/bro/download/ip/tag1&&!tag2'; ?></pre>
+    <pre><?php echo $baseurl.'/attributes/zeek/download/ip/tag1&&!tag2'; ?></pre>
```
Comment (Rafiot) [inline app/View/Events/legacy_automation.ctp:220]: Alias needed.

File: app/View/Events/legacy_automation.ctp [hunk 10]
```diff
@@ -238,10 +238,10 @@
     ?></pre>
     <code>&lt;request&gt;&lt;type&gt;ip&lt;/type&gt;&lt;eventid&gt;!51&lt;/eventid&gt;&lt;eventid&gt;!62&lt;/eventid&gt;&lt;withAttachment&gt;false&lt;/withAttachment&gt;&lt;tags&gt;APT1&lt;/tags&gt;&lt;tags&gt;!OSINT&lt;/tags&gt;&lt;from&gt;false&lt;/from&gt;&lt;to&gt;2015-02-15&lt;/to&gt;&lt;/request&gt;</code><br /><br />
     <p><?php echo __('Alternatively, it is also possible to pass the filters via the parameters in the URL, though it is highly advised to use POST requests with JSON objects instead. The format is as described below');?>:</p>
-    <pre><?php echo $baseurl.'/attributes/bro/download/[type]/[tags]/[event_id]/[allowNonIDS]/[from]/[to]/[last]'; ?></pre>
-    <b>type</b>: <?php echo __('The Bro type, any valid Bro type is accepted. The mapping between Bro and MISP types is as follows');?>:<br />
+    <pre><?php echo $baseurl.'/attributes/zeek/download/[type]/[tags]/[event_id]/[allowNonIDS]/[from]/[to]/[last]'; ?></pre>
```
Comment (Rafiot) [inline app/View/Events/legacy_automation.ctp:241]: Alias needed.

**Review by Rafiot — COMMENTED** [review]
Marking the places that could break the existing scripts.

@iglocska am I missing some?

**Comment by SteveClement** [comment]
Bump @iglocska, is this all good?

**Comment by Rafiot** [comment]
It's not ready for prime time yet, we may do it differently.

**Pull request closed without merging**